.github/workflows/test-e2e.yaml

```diff
@@ -0,0 +1,71 @@
+name: E2E Test
+
+on:
+  pull_request
+
+jobs:
+  e2e-test:
+    name: E2E Test
+    runs-on:
+      labels: ubuntu-latest-16-cores
```

Comment:
Similar to this PR, please can we migrate to OCI runners ?
https://github.com/kubeflow/trainer/pull/2739